robosistance/src/models/robot_model.rs

```diff
@@ -0,0 +1,9 @@
+use mongodb::bson::oid::ObjectId;
```

Comment:
I think this file would fit fine in the db folder, I don't think you need another one.